unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/EventsSubSection/EventsSubSectionMenu/EventsSubSectionComponentController.cs

```diff
@@ -0,0 +1,73 @@
+using DCL;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface IEventsSubSectionComponentController
```

Comment:
Use `IDisposable` here.

Reply:
Correct! Done in https://github.com/decentraland/unity-renderer/pull/1238/commits/758ac9b508e15b6c9518db4a59c59f4338c26b21. Thanks!